tests/appsec/test_processor.py

```diff
@@ -200,6 +202,87 @@ def test_appsec_body_no_collection_snapshot(tracer):
         assert "triggers" in json.loads(span.get_tag(APPSEC_JSON))
 
 
+def test_ip_block(tracer):
+    with override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)), override_global_config(dict(_appsec_enabled=True)):
+        _enable_appsec(tracer)
+        with _asm_context.asm_request_context_manager("8.8.4.4", {}):
+            with tracer.trace("test", span_type=SpanTypes.WEB) as span:
+                set_http_meta(
+                    span,
+                    Config(),
+                )
+
+            assert "triggers" in json.loads(span.get_tag(APPSEC_JSON))
+            assert _context.get_item("http.request.remote_ip", span) == "8.8.4.4"
+            assert _context.get_item("http.request.blocked", span)
+            assert "block" in _context.get_item("http.request.waf_actions", span)
+
+
+def test_ip_not_block(tracer):
+    with override_env(dict(DD_APPSEC_RULES=RULES_GOOD_PATH)), override_global_config(dict(_appsec_enabled=True)):
+        _enable_appsec(tracer)
+        with _asm_context.asm_request_context_manager("8.8.8.4", {}):
```

Comment:
```suggestion
        with _asm_context.asm_request_context_manager("1.1.1.1", {}):
```

or please add a comment or something... I stared at this and `test_ip_block` for about 5 minutes before I was able to spot the difference between the two tests.

or maybe have some constants at top like `BLOCKED_IP = "8.8.4.4"; GOOD_IP = "8.8.8.4"` so make these tests more legible.

Reply:
Done